include/openPMD/IO/AbstractIOHandlerImplCommon.hpp

```diff
@@ -78,11 +78,15 @@ class AbstractIOHandlerImplCommon : public AbstractIOHandlerImpl
     /**
      * Get the writable's containing file.
      * @param writable The writable whose containing file to figure out.
+     * @param preferParentFile If true, the file is set to the parent's file if
+     *     present. Otherwise, the parent file is only considered if no own file
+     *     is defined.
```

Comment:
@franzpoeschel shall we add a typical use case here?
```suggestion
     *     is defined. This is usually needed when switching between iterations when opening paths.
```

Reply:
I'll add this already in :)